main.go

```diff
@@ -144,6 +144,7 @@ func main() {
 		log.Error("Failed to create SubnetManager: ", err)
 		os.Exit(1)
 	}
+	log.Info(fmt.Sprintf("Created subnet manager: %+v", sm))
```

Comment:
weird, I still see the old log format on your remote branch: https://github.com/t0mmyt/flannel/blob/awsvpc-multiaz-rebase/main.go#L147